app/domain/logs.rb

```diff
@@ -14,6 +14,26 @@ module Conjur
       code: "CONJ00038I"
     )
 
+    FetchingTrustedProxies = ::Util::TrackableLogMessageClass.new(
+        msg:  "Fetching trusted proxies list from {0-host-id}",
+        code: "CONJ00039D"
+    )
+
+    FetchedTrustedProxies = ::Util::TrackableLogMessageClass.new(
+        msg:  "{0-list-length} trusted proxies ip addresses has fetched",
+        code: "CONJ00040D"
+    )
+
+    DeletingTrustedProxiesDuplications = ::Util::TrackableLogMessageClass.new(
+        msg:  "Deleting duplications from trusted proxies list",
+        code: "CONJ00041D"
+    )
+
+    TrustedProxiesAmount = ::Util::TrackableLogMessageClass.new(
+        msg:  "Trusted proxies list contains {0-list-length} ip addresses",
```

Comment:
Trusted proxies list contains {0-list-length} IP addresses

(IP is always uppercase)

Reply:
Fixed :)